### DISCUSSION TITLE
Option to suppress names of streets, places, etc alongside of a route.

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#20039] [Ideas]
Hello,

I suggest an option that suppresses displaying names of streets, places, buildings, etc, while following a route, especially when travelling with greater speed than a bicycle. For example: when I drive along an "Autobahn" (motorway, highway), I do not really need to be shown the names of streets, buildings and things that are near my route on the road. 

Not showing these information will use less computing power (I guess), and it will decrease the distraction when looking at the screen.

Of course, such an option has to be profile dependent, because different kind of information are required depending on the type of vehicle in use. On a bicycle I am very much interested in nearby street names, but not when driving 130 km/h on a fast lane.

**Comment by yuriiurshuliak** [comment]
We currently have similar functionality. For example, you can hide the following objects:
Boundaries, Polygons, Buildings, Hide water, House numbers, [object Object], POI icons, POI labels, Underground objects, Overground objects.

You can find more detailed information at this link:
https://osmand.net/docs/user/map/vector-maps/#hide

**Comment by sjvudp** [comment]
It should be a setting (like for TomTom). In general I prefer seeing street names, though.
